### PR TITLE
added mycart

### PR DESCRIPTION
**Pull request by sushmita12345**
Added myCart page to the E-commerce website.

Sample
Screenshot 2022-02-11 at 2 04 46 AM

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
looks clean to me. good to go

File: assets/css/myCart.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+.quantity-counter {
+  height: 1.5rem;
+  width: 1.5rem;
+  border-radius: 50%;
+  padding-top: 0.2rem;
+  border: 1px solid var(--grey);
+  background-color: var(--white);
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
```
Comment (PuneetP16) [inline assets/css/myCart.css]: could use pixels instead.

**Review by PuneetP16 — APPROVED** [review]
GTM

File: assets/css/myCart.css (hunk 1 above)
Comment (sushmita12345) [reply assets/css/myCart.css]: Ok, I'll update.

**Pull request merged**